packages/ui/src/composites/wui-all-wallets-image/index.ts

```diff
@@ -1,31 +1,39 @@
 import { html, LitElement } from 'lit'
-import { customElement, property } from 'lit/decorators.js'
+import { customElement, property, state } from 'lit/decorators.js'
 import { globalStyles } from '../../utils/ThemeUtil'
 import '../wui-wallet-image'
 import styles from './styles'
-
-export interface WalletImage {
-  src: string
-  walletName: string
-}
+import type { IWalletImage } from '../../utils/TypesUtil'
 
 @customElement('wui-all-wallets-image')
 export class WuiAllWalletsImage extends LitElement {
   public static styles = [globalStyles, styles]
 
   // -- state & properties ------------------------------------------- //
-  @property({ type: Array }) public walletImages: WalletImage[] = []
+  @property({ type: Array }) public walletImages: IWalletImage[] = []
+
+  @state() public totalImages = 4
```

Comment:
We should try to avoid using state in ui package, you can define constants outside of class via simple `const TOTAL_IMAGES = 4` or if you need to latter update this value, you can make it a private class one i.e. `private totalImages = 4`.

State should be used only when we need to trigger component's re-render from it self, which in case of ui we want to avoid, because we want scaffold package to control this logic through `@properties`